packages/commons-ui-core/src/utils/transformDocMediaUrl.js

```diff
@@ -0,0 +1,13 @@
+const transformDocMediaUrl = (doc) => {
```

Comment:
`payload`-related functionality should **not** be in `@commons-ui/core` or `@commons-ui/next` packages @koechkevin .